keps/sig-auth/3299-kms-v2-improvements/README.md

```diff
@@ -99,7 +96,7 @@ It further proposes a SIG-Auth maintained KMS plugin reference implementation. T
 ## Proposal
 
 Performance, Health Check, Observability and Rotation:
-- Support key hierarchy in KMS plugin that generates local KEK
+- Support re-using DEK when the KEK key ID is stable
```

Comment:
do we need to remove `can contain the encrypted local KEK,`?

https://github.com/kubernetes/enhancements/pull/3952/files#diff-7ce75bf08575b191bd53778b1125e122bfc27c40603cb8462d9155591fd9cc6eR104